### ISSUE TITLE
I have one question...

### ISSUE THREAD

**Issue by Contrabassee** [NVlabs/stylegan2-ada-pytorch#304]
I saw that there were train set, validation set, test set in the paper figure1.

I don't know what they are in the code. I don't think there is ...validation set and test set

and that metrics(FID,PPL etc.) are about train set??

plz..i wait for your comments!
